class error_creeaza(Exception):
    pass

class error_adaugare(Exception):
    pass
